.devcontainer/devcontainer.json

```diff
@@ -1,36 +1,34 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
-// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.0/containers/javascript-node
+// https://github.com/devcontainers/images/tree/main/src/javascript-node
 {
   "name": "Next.js Dev Container",
   "build": {
     "dockerfile": "Dockerfile",
-    // Update 'VARIANT' to pick a Node version: 18, 16, 14.
-    // Append -bullseye or -buster to pin to an OS version.
-    // Use -bullseye variants on local arm64/Apple Silicon.
+    // Update 'VARIANT' to pick a Node version: 22, 20, 18.
+    // Append -bookworm or -bullseye to pin to an OS version.
     "args": {
-      "VARIANT": "20-bullseye"
+      "VARIANT": "22-bookworm"
     }
   },
   // Configure tool-specific properties.
   "customizations": {
     // Configure properties specific to VS Code.
     "vscode": {
       // Add the IDs of extensions you want installed when the container is created.
-      "extensions": ["dbaeumer.vscode-eslint"]
+      "extensions": ["esbenp.prettier-vscode"]
```

Comment:
"dbaeumer.vscode-eslint" isn't necessary because [Node.js & JavaScript](https://github.com/devcontainers/images/tree/main/src/javascript-node) has already installed that.
Then I add a [prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) instead of that.